src/lib/alarmMgr/alarmMgr.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include "common/limits.h"
+#include "alarmMgr/AlarmManager.h"
+
+
+
+/* ****************************************************************************
+*
+* alarmMgr - 
+*/
+AlarmManager alarmMgr(ALARM_MGR_NOTIFICATION_ERROR_LOG_INTERVAL, ALARM_MGR_BAD_INPUT_LOG_INTERVAL);
```

Comment:
Another strategy is to create an instance of the class (see example at contextBrokerInit() for Notifier singleton, https://github.com/telefonicaid/fiware-orion/blob/6bca41e5f40f71f14ad52d3b91394ae0494deec6/src/app/contextBroker/contextBroker.cpp#L1297) but it is also ok this way.

NTC